index.rst

```diff
@@ -1310,25 +1238,23 @@ parole, il contributo fornito dal singolo al conseguimento della
 
 Le dimensioni che compongono la *performance* individuale sono:
 
--  ***risultati**,* riferiti agli obiettivi *annuali* inseriti nel Piano
-   della *performance* o negli altri documenti di programmazione;
+-  \*\ **risultati**,\* riferiti agli obiettivi *annuali* inseriti nel
```

Comment:
Questo asterisco è rimasto

Reply:
potrebbe non trattarsi di un errore. il markup `***risultati**,*` tecnicamente non è RST corretto. negli altri casi pandoc fa comunque il parsing come vogliamo noi, in questo caso riconosce il grassetto interno e considera gli asterischi addizionali come semplice testo. non è male, non sono sicuro che saprei immaginare un modo migliore di gestire casi simili. vedo che anche il parser di Github fa lo stesso. comunque se sei d'accordo non investirei tempo su questo, a meno che non si ripresenti più volte in futuro

Reply:
@danse 
Non avevo notato che la virgola è tra due asterischi. Certamente mi sembra un caso patologico su cui non vale la pena perderci troppo tempo. Lo posso correggere io a mano successivamente.